src/EditableDialogBox/EditableItem.php

```diff
@@ -5,14 +5,16 @@
 
 class EditableItem extends DialogBoxItem
 {
+    public const TYPE = 'editable';
+
     private string $name;
     private string $label = '';
     /** @var array<string, bool|float|int|string> */
     private array $config = [];
 
-    public function __construct(string $type, string $name)
+    public function __construct(string $name)
     {
-        parent::__construct($type);
+        parent::__construct();
```

Comment:
I don't see any advantage in having to call a constructor without arguments that will set a value based on a constant that you have to override. For me, this is too much "magic".